### PR TITLE
Add listen count card

### PR DESCRIPTION
**Pull request by shivam-kapila**
Please don't merge till UI release.

Preview:
image

### CODE REVIEW

**Comment by paramsingh** [comment]
>Please don't merge till UI release.

Why not? This isn't used anywhere, when are you planning on getting this merged?

**Comment by shivam-kapila** [comment]
> Why not? This isn't used anywhere, when are you planning on getting this merged?
Ah sorry. I thought I pushed the code for implementation too. Please review and let me know of any other tests we need.

File: listenbrainz/webserver/static/js/src/listens/ListenCountCard.test.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import ListenCountCard, { ListenCountCardProps } from "./ListenCountCard";
+
+const props: ListenCountCardProps = {
+  listenCount: 100,
+};
+
+describe("Listen Count Card", () => {
+  it("renders correctly", () => {
+    const wrapper = mount(<ListenCountCard {...props}>Test</ListenCountCard>);
```
Comment (paramsingh) [inline listenbrainz/webserver/static/js/src/listens/ListenCountCard.test.tsx:12]:
```suggestion
    const wrapper = mount(<ListenCountCard {...props} />);
```

File: listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  return (
+    <>
+      <Card id="listen-count-card">
+        <h4>
+          You have listened to
+          <hr />
+        </h4>
+        <div>
+          <h4>{listenCount || `&nbsp;`}</h4>
```
Comment (paramsingh) [inline listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx:19]: looking at this again, would this not render the number 0?

Comment (shivam-kapila) [reply listenbrainz/webserver/static/js/src/listens/ListenCountCard.tsx:19]: Didnt Get you

**Pull request merged**